webapi/src/main/twirl/queries/sparql/v1/generateInsertStatementsForCreateValue.scala.txt

```diff
@@ -59,15 +59,21 @@
         @value match {
 
             case textValueSimple: TextValueSimpleV1 => {
```

Comment:
I think this logic has to be added to `src/main/twirl/queries/sparql/v1/addValueVersion.scala.txt` too.

Reply:
Or can the language only be added when creating new values using the bulk import?

Reply:
no, it is not supposed to work just with bulk import. This logic is currently implemented for creation of TextValue for first time both for bulk import requests and individual resource creation requests. You are right, the same logic can be added in case a TextValue already exists and a new version of it should be added with specified language.

Reply:
So some logic needs to be added to

- `ResourcesResponderV1`
- `ValuesResponderV1`

I can help you with that if necessary.